glide-core/redis-rs/redis/src/aio/multiplexed_connection.rs

```
@@ -582,8 +582,7 @@ impl MultiplexedConnection {
                 }
             }
         }
-        // TODO: remove this when `raise_on_error` flag will be added
-        let value = result.and_then(|v| v.extract_error())?;
+        let value = result?;
         match value {
             Value::Array(mut values) => {
```

```rs
match result {
    Ok(Value::Array(mut values) => {
...
```

---

why is that? we can raise an error if value is a redis-error (like a parsing problem that occured)